.gitignore

```diff
@@ -2,4 +2,5 @@
 .vscode
 .pytest_cache
 *__pycache__
-giant_contact.egg-info
\ No newline at end of file
+giant_contact.egg-info
+poetry.lock
```

Comment:
How come this is ignored?

Reply:
It was Jon's thought process. Because it only ever lives as a dependency and is never run externally you wouldn't need the `.lock` file